internal/testrunner/runners/pipeline/test_result.go

```diff
@@ -186,6 +186,22 @@ func adjustTestResult(result *testResult, config *testConfig) (*testResult, erro
 			}
 		}
 
+		if skipGeoIP {
+			// remove common related geoIP keys
+			geoIPKeys := []string{
+				"source.as",
+				"source.geo",
+				"destination.as",
+				"destination.geo",
```

Comment:
Could we get a more complete list from ECS?

Reply:
Updated this list with the fields that I found related to geo from here
https://github.com/elastic/ecs/blob/18bb1e585a6b24ce2a35ebed49af361d8dfbc585/generated/ecs/ecs_flat.yml

This list is based on looking for fields like `*.geo.country_name` or `*.as.organization.name` as a reference.